### DISCUSSION TITLE
USB sticks not automounted?

### DISCUSSION THREAD

**Discussion by tve** [Manjaro-Sway/manjaro-sway#357] [Q&A] [answered]
When I plug a USB stick in it doesn't get automounted. I know there are security issues to consider, but it's really convenient (plus the most insiduous damage can be done just with plugging in, mounting is not needed...).

Is the lack of such automount support intentional? I know it's there by default in the Cinnamon and KDE editions. I don't know how they do it but I can imagine it starts by dragging umpteen dependencies in... I do see scripts to do it just using systemd.

(Just to clarify, the devices get detected just fine at the USB level and show up as /dev/sdX. I can do something like `sudo mount /dev/sdX /mnt` and it works. I'm missing the nicety of having it appear as `/run/user/tve/GARMIN` or `/run/user/tve/my-stick`, which is really useful when dealing with scripts to pull data off a sports tracker or put data onto sdcards one swaps all day long with an rPi...)

**Comment by boredland** [comment, marked as answer]
I think this worked at some point and currently doesn't. That's a bug.

**Reply by tve** [reply]
can you give me some keywords on how it's supposed to work? Then I can troubleshoot...

**Reply by boredland** [reply]
pcmanfm has a deamon for that and we don't start it rn. fix is on its way.

**Reply by boredland** [reply]
a new version 12.1.2 should be out in a few minutes. I tried it and for me auto-mount is working nicely again.
